pstress/generate_mysqld_options.sh

```diff
@@ -0,0 +1,511 @@
+    echo "${OPTION}=40" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--lc-messages" ]; then
+    echoit "  > Adding possible values en_US for option '${OPTION}' to the final list..."
+    echo "${OPTION}=en_US" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--lc-time-names" ]; then
+    echoit "  > Adding possible values en_US for option '${OPTION}' to the final list..."
+    echo "${OPTION}=en_US" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--master-info-file" ]; then
+    echoit "  > Adding possible values master.info for option '${OPTION}' to the final list..."
+    echo "${OPTION}=master.info" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--mysqlx-compression-algorithms" ]; then
+    echoit "  > Adding possible values DEFLATE_STREAM, LZ4_MESSAGE, ZSTD_STREAM for option '${OPTION}' to the final list..."
+    echo "${OPTION}=DEFLATE_STREAM" >> ${OUTPUT_FILE}
+    echo "${OPTION}=LZ4_MESSAGE" >> ${OUTPUT_FILE}
+    echo "${OPTION}=ZSTD_STREAM" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--optimizer-switch" ]; then
+    echoit "  > Adding possible values batched_key_access, block_nested_loop, condition_fanout_filter for option '${OPTION}' to     the final list..."
+    echo "${OPTION}=index_merge=on,index_merge_union=on,index_merge_sort_union=on,index_merge_intersection=on,engine_condition_p    ushdown=on,index_condition_pushdown=on,mrr=on,mrr_cost_based=on,block_nested_loop=on,batched_key_access=off,materialization=    on,semijoin=on,loosescan=on,firstmatch=on,duplicateweedout=on,subquery_materialization_cost_based=on,use_index_extensions=on    ,condition_fanout_filter=on,derived_merge=on,use_invisible_indexes=off,skip_scan=on,hash_join=on,subquery_to_derived=off,pre    fer_ordering_index=on,hypergraph_optimizer=off,derived_condition_pushdown=on,favor_range_scan=off" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--optimizer-trace-features" ]; then
+    echoit "  > Adding possible values greedy_search , range_optimizer , dynamic_range , epeated_subselect for option '${OPTION}    ' to the final list..."
+    echo "${OPTION}=greedy_search=on,range_optimizer=on,dynamic_range=on,repeated_subselect=on" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--protocol-compression-algorithms" ]; then
+    echoit "  > Adding possible values zlib, zstd, uncompressed for option '${OPTION}' to the final list..."
+    echo "${OPTION}=zlib" >> ${OUTPUT_FILE}
+    echo "${OPTION}=zstd" >> ${OUTPUT_FILE}
+    echo "${OPTION}=uncompressed" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--relay-log-info-file" ]; then
+    echoit "  > Adding possible value relay-log.info for option '${OPTION}' to the final list..."
+    echo "${OPTION}=relay-log.info" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--replica-exec-mode" ]; then
+    echoit "  > Adding possible value STRICT and IDEMPOTENT for option '${OPTION}' to the final list..."
+    echo "${OPTION}=STRICT" >> ${OUTPUT_FILE}
+    echo "${OPTION}=IDEMPOTENT" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--replica-load-tmpdir" ]; then
```

Comment:
Exclude this option and remove the elif block

Reply:
Fixed

Reply:
removed --replica-load-tmpdir